python/lsst/ip/isr/isrTask.py

```diff
@@ -1916,14 +1916,17 @@ def defineEffectivePtc(self, ptcDataset, detector, bfGains, overScans, metadata)
             doWarningPtcValidation = False
 
             metadata[f"LSST GAIN {amp.getName()}"] = effectivePtc.gain[ampName]
+            metadata[f"LSST ISR GAIN {amp.getName()}"] = effectivePtc.gain[ampName]
```

Comment:
Probably should reorganize these to pair the old header values (LSST GAIN / LSST READNOISE) with a comment that these are here for compatibility.